staging/src/k8s.io/apiserver/pkg/apis/config/OWNERS

```diff
@@ -0,0 +1,7 @@
+approvers:
```

Comment:
the OWNERS file from `k8s.io/apiserver` would probably be a more appropriate starting point

Reply:
@luxas @sttts what's your opinion?

Reply:
+1 for @liggitt's suggestion.